config/software/automake.rb

```diff
@@ -15,21 +15,18 @@
 #
 
 name "automake"
-default_version "1.11.2"
+default_version "1.16"
```

Comment:
Looks like this will require using a newer version of autoconf which I'll fix here shortly. I'll also do some additional tests:
```
    configure.ac:18: error: Autoconf version 2.69 or higher is required
```